pom.xml

```diff
@@ -70,6 +70,7 @@
     <module>tracing</module>
     <module>edge</module>
     <module>metrics</module>
+    <module>dynamic-config</module>
```

Comment:
create another root modules?
maybe it's can a child of foundations like foundation-cc?

Reply:
Creating this new root module because considering that foundation module is too basic for users to change between foundation-config-cc, foundation-config-apollo or others. maybe we could create this module for all dynamic config sources including foundation-config-cc?

Thanks @wujimin and @eric-lee-ltk for all the comments above.

Reply:
ok, you can create a new issue to trace this.

Reply:
Issues have been created at https://issues.apache.org/jira/browse/SCB-208